Sample/Vungle Sample App/BannerList.m

```diff
@@ -0,0 +1,139 @@
+//
+//  BannerList.m
+//  Vungle Sample App
+//
+//  Created by John Mai on 2/6/20.
+//  Copyright © 2020 Jordyn Chuhaloff. All rights reserved.
+//
+
+#import "BannerList.h"
+#import "Constants.h"
+#import "DisplayTableViewCell.h"
+
+@interface BannerList () {
+    CGFloat screenHeight;
+    CGFloat screenWidth;
+}
+@property (strong, nonatomic) IBOutlet UIView *view;
+@property const int totalSize;
+@property const int AD_PER_ITEM;
+@property NSMutableArray* adArr;
+@property (nonatomic, strong) VungleSDK *sdk;
+@end
+
+@implementation BannerList
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    self.adArr = [[NSMutableArray alloc] init];
+    self.totalSize = 40;
+    self.AD_PER_ITEM = 4;
+    [self startVungle];
+    [self.view addSubview:self.tableView];
+    self.tableView.delegate = self;
+    self.tableView.dataSource = self;
+}
+-(void)viewDidAppear:(BOOL)animated {
+    [self.tableView reloadData];
+}
+-(NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    return 1;
+}
+- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath {
+    
+    CGFloat height = 50.0;
+    if (indexPath.row == (self.totalSize - 4)) {
+        height = 250.0;
+    }
+    return height;
+}
+-(UITableViewCell *)tableView:(UITableView*) tableView cellForRowAtIndexPath:(nonnull NSIndexPath *)indexPath {
+    NSError *error = nil;
+    UITableViewCell *cell;
+        if ((indexPath.row == 0) || (indexPath.row == (self.totalSize / 2)) || (indexPath.row == 1)) {
+           
+            cell = [self.tableView dequeueReusableCellWithIdentifier:@"adCell" forIndexPath:indexPath];
```

Comment:
Here - why not dequeue into a `DisplayTableViewCell *bannerCell` directly? And, at the end of the block after line 60, `return bannerCell;`?